src/test/java/net/imagej/ops/AbstractOpTest.java

```diff
@@ -240,14 +240,14 @@ private int pseudoRandom() {
 		assertFalse("More elements than expected", a.hasNext());
 	}
 
-	public <T extends RealType<T>> boolean areCongruent(final IterableInterval<T> in, final RandomAccessible<T> out){
+	public <T extends RealType<T>> boolean areCongruent(final IterableInterval<T> in, final RandomAccessible<T> out, final double epsilon){
 		Cursor<T> cin = in.localizingCursor();
 		RandomAccess<T> raOut = out.randomAccess();
 		
 		while(cin.hasNext()){
 			cin.fwd();
 			raOut.setPosition(cin);
-			if(cin.get().getRealDouble() != raOut.get().getRealDouble()) return false;
+			if((cin.get().getRealDouble() - raOut.get().getRealDouble()) > epsilon) return false;
```

Comment:
You will have to check the absolute difference to support the case where `cin.get().getRealDouble()` is less than `raOut.get().getRealDouble()` assuming that epsilon will always be positive which is what I would suggest.